### PR TITLE
Add sanetize->sanitize

### PR DESCRIPTION
**Pull request by sshane**
| sanetize | sanetized | sanetizer | sanetizing |
| :------------- | :------------- | :------------- | :------------- |
| [sanetize=1k files](https://github.com/search?q=sanetize&type=code)  | [sanetized=362 files](https://github.com/search?q=sanetized&type=code)  | [sanetizer=77 files](https://github.com/search?q=sanetizer&type=code) | [sanetizing=25 files](https://github.com/search?q=sanetizing&type=code) | 
| [sanitize=5.8m files](https://github.com/search?q=sanitize&type=code)  | [sanitized=1.2m files](https://github.com/search?q=sanitized&type=code)  | [sanitizer=1.6m files](https://github.com/search?q=sanitizer&type=code) | [sanitizing=144k files](https://github.com/search?q=sanitizing&type=code) |

### CODE REVIEW

File: codespell_lib/data/dictionary.txt [hunk 1]
```diff
@@ -47695,6 +47695,8 @@ sandwitch->sandwich, sand witch,
 sandwitched->sandwiched
 sandwitches->sandwiches, sand witches,
 sandwitching->sandwiching
+sanetize->sanitize
+sanetizing->sanitizing
```
Comment (sshane) [inline codespell_lib/data/dictionary.txt:47699]:
```suggestion
sanetizing->sanitizing
sanetized->sanitized
sanetizer->sanitizer
```

File: codespell_lib/data/dictionary.txt [hunk 2]
```diff
@@ -47695,6 +47695,10 @@ sandwitch->sandwich, sand witch,
 sandwitched->sandwiched
 sandwitches->sandwiches, sand witches,
 sandwitching->sandwiching
+sanetize->sanitize
+sanetized->sanitized
+sanetizer->sanitizer
+sanetizing->sanitizing
```
Comment (sshane) [inline codespell_lib/data/dictionary.txt:47701]:
```suggestion
sanetize->sanitize
sanetized->sanitized
sanetizer->sanitizer
sanetizing->sanitizing
```

**Comment by DimitriPapadopoulos** [comment]
Time permitting, could you add also `sanetizes` and `sanetizers`?

Adding British spelling variant would be great:
```
sanetise->sanitise
```

**Review by DimitriPapadopoulos — APPROVED** [review]

**Comment by DimitriPapadopoulos** [comment]
Thank you @sshane.

**Pull request merged**